lib/color_filter.dart

```diff
@@ -11,7 +11,8 @@ class ColorFilter {
   /**
    * Makes the color lighter by the percentage specified in the first argument, or by 10% if no percentage is specified. Percentages should be specified as a float, e.g. an argument of 0.25 will result in a color 25% lighter than the original. The lightening conversion is performed by adjusting the y component of the color in XYZ color space.
    */
-  static ColorFilter lighten = new ColorFilter((CielabColor color, [List args]) {
+  static ColorFilter lighten = new ColorFilter((Color inputColor, [List args]) {
+    CielabColor color = inputColor.toCielabColor();
```

Comment:
These `.toCielabColor()` calls will basically be a `return this` since the passed in inputColor will already be a CielabColor